src/reconcile/rec_visibility.c

```diff
@@ -368,9 +371,9 @@ __wt_rec_upd_select(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins, v
                 upd_select->start_txn = upd->txnid;
             }
 
-            /* Use stop timestamp as durable timestamp if exist. */
-            if (upd_select->stop_ts != WT_TS_MAX)
-                upd_select->durable_ts = upd_select->stop_ts;
+            /* Use tombstone durable timestamp as overall durable timestamp if exist. */
```

Comment:
I suggest rewording as: "Use the tombstone durable timestamp as the overall durable timestamp if it exists."

Reply:
Fixed as suggested.